### PR TITLE
adding support for dropdown labels

### PR DESCRIPTION
**Pull request by amberRrucker**
@bighappyface This should work for the ask you had for adding support for drop down labels. You can see it on Managed Offerings. Let me know if you need anything else for this one!

### CODE REVIEW

**Comment by bighappyface** [comment]
+1

File: styleguide/derek/_partials/_mainnav.ejs [hunk 1]
```diff
@@ -109,7 +109,7 @@
               <li class="navbar-topItem">
                 <span class="navbar-topLink dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Solutions</span>
                 <ul class="dropdown-menu navbar-dropDownMenu">
-                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLink active">Managed Offerings</a></li>
+                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLabel">Managed Offerings</a></li>
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_mainnav.ejs]: @amberRrucker Can you switch the `<a>` to a `<span>` here, so it doesn't act like a link in the browser.

Comment (amberRrucker) [inline styleguide/derek/_partials/_mainnav.ejs]: Fixed it!

File: styleguide/derek/_partials/_mainnav.ejs [hunk 2]
```diff
@@ -109,10 +109,10 @@
               <li class="navbar-topItem">
                 <span class="navbar-topLink dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Solutions</span>
                 <ul class="dropdown-menu navbar-dropDownMenu">
-                  <li class="navbar-dropDownItem"><a href="javascript:void(0)" column="1" columntitle="1" class="navbar-dropDownLink active">Managed Offerings</a></li>
+                  <li class="navbar-dropDownItem"><label column="1" columntitle="1" class="navbar-dropDownLabel">Managed Offerings</a></li>
```
Comment (WidgetsBurritos) [inline styleguide/derek/_partials/_mainnav.ejs]: I'm not entirely sure that is the proper use of the `<label>` tag.  That's for connecting with input fields.  That's why I recommending using the '<span>' tag.  But even if we do stick with that, you have a closing '</a>' tag, not a '</label>'.

Comment (amberRrucker) [inline styleguide/derek/_partials/_mainnav.ejs]: omg i am so sorry. I wasn't paying nearly as much attention as I should have. Fixing it right now.

**Comment by WidgetsBurritos** [comment]
👍

**Comment by bighappyface** [comment]
:shipit:

**Pull request merged**